includes/actions/python/check-upload-publish-packages/action.yaml

```diff
@@ -47,7 +47,7 @@ runs:
         python -m zipfile -t $WHEEL
         python -m zipfile -l $WHEEL
         echo
-        if [ "$(uname)" = "Linux" -a $PYVER -eq 3 ]; then
+        if [ "$(uname)" = "Linux" -a $PYVER -eq 3 -a "${WHEEL: -8}" = "-any.whl" ]; then
```

Comment:
Do we want to check "equal" or "not equal"?

Reply:
right, I'll fix it

Reply:
fixed